.terraform/helpers/generated-main.template.tf

```diff
@@ -0,0 +1,48 @@
+# Auto-generated by generate-config.sh
```

Comment:
What if we don't add generated files. Would this PR becomes much smaller?

Reply:
These three 'template' files aren't actually generated -- they're used to copy and then append to the "actual" generated files which are not in git.